frontend/src/Profile/ProfileContents.js

```diff
@@ -0,0 +1,105 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import {withStyles} from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import Link from '@material-ui/core/Link';
+
+const useStyles = (theme) => ({
+  large: {
+    width: theme.spacing(7),
+    height: theme.spacing(7),
+  },
+  cardStyle: {
+    display: 'block',
+    width: '50vw',
+    height: '50vw',
+  },
+  cardItems: {
+    paddingTop: 25,
+    paddingLeft: 50,
+  },
+});
+
+/**
+   * Renders profile info
+   * @return {Grid} - Returns Grid of cards
+   */
+class ProfileContents extends Component {
+  /**
+   * Sets up props to be used in components
+   * Should add more state fields later when we have a route for profile info
+   * @param {*} props
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      email: '',
+    };
+  }
+  /**
+   * Function needed for eslint to be happy
+   */
+  static get propTypes() {
+    return {
+      classes: PropTypes.any,
+    };
+  }
+  /**
```

Comment:
`classes: PropTypes.shape({
        cardItems: PropTypes.style,
        cardStyle: PropTypes.style,
        large: PropTypes.style,
      })`

PropTypes.shape can be used to describe objects and 'classes' is an object containing the styles from useStyles so this type description might fit better.